src/test/java/testsuite/integration/utility/ContainerHelper.java

```diff
@@ -143,13 +143,12 @@ protected Long execInContainer(InspectContainerResponse containerInfo, Consumer<
       dockerClient.execStartCmd(execCreateCmdResponse.getId()).exec(callback).awaitCompletion();
     }
 
-    final Long exitCode = dockerClient.inspectExecCmd(execCreateCmdResponse.getId()).exec().getExitCodeLong();
-    return exitCode;
+    return dockerClient.inspectExecCmd(execCreateCmdResponse.getId()).exec().getExitCodeLong();
   }
 
   protected boolean isRunning(InspectContainerResponse containerInfo) {
     try {
-      return containerInfo != null && containerInfo.getState().getRunning();
+      return containerInfo != null && Boolean.TRUE.equals(containerInfo.getState().getRunning());
```

Comment:
I can't believe that analysis suggests that...

Reply:
Should I change it back?

Reply:
Yes please